src/system_sensors.py

```diff
@@ -177,6 +179,11 @@ def get_wifi_strength():  # check_output(["/proc/net/wireless", "grep wlan0"])
 def get_rpi_power_status():
     return _underVoltage.get()
 
+def get_host_name():
+    return socket.gethostname()
+
+def get_host_ip():
+    return socket.gethostbyname(socket.gethostname())
```

Comment:
This returns 127.0.1.1 on Raspbian when the hostname is changed by using the raspi-config tool(this puts the hostname in the /etc/hosts file)

Reply:
Good spotting. I'll look into it.